src/components/common/PeripheralsDialog.vue

```diff
@@ -368,7 +368,8 @@ export default class ManualProbeDialog extends Mixins(StateMixin) {
         type: 'can',
         count: this.canbusUuids
           ? Object.values(this.canbusUuids)
-            .reduce((a, b) => a + b.length, 0)
+            .filter(value => Array.isArray(value))
+            .reduce((a, b) => a + (b ? b.length : 0), 0)
```

Comment:
If the filter for `Array.isArray()` is done, this should not be required as `b` will definitely be defined and an array.

Reply:
I encountered a problem once in the test that `b` does not have the attribute `length`. However, it was not encountered in subsequent tests. This change may not be required.

Reply:
I just tested it and it is indeed like this, so I will restore these changes.